packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -207,7 +172,10 @@ class MemoryBodyState extends State<MemoryBody>
     addAutoDisposeListener(controller.androidChartVisibleNotifier, () {
       setState(() {
         if (controller.androidChartVisibleNotifier.value) {
-          MemoryScreen.gaAction(key: MemoryScreen.androidChartButtonKey);
+          ga.select(
```

Comment:
maybe add a helper to DevTools screen that 
always passes in the analytics constant for the current screen?

Reply:
This is tricky because the calls to `ga.select` do not happen in the Widget subclasses of Screen. At the time that we call `ga.select`, we often times do not even have access to the Screen object, which is why we would have to get the id from a static method on the screen class, but we don't have access to the screenId from a screen in a static context.